tox.ini

```diff
@@ -36,7 +39,7 @@ description =
 # The following provides some specific pinnings for key packages
 deps =
     oldestdeps: numpy==1.17.*
-    devdeps: :NIGHTLY:numpy
```

Comment:
We should also remove the `indexserver` entry on line 11.

Reply:
done